### PR TITLE
NagiosPB : Fix nagios server linter

### PR DESCRIPTION
**Pull request by steelhead31**
Amend the nagios server playbook to exclude the vault file by default, but update the instructions to include a section on re-adding the line, after the new vault has been created ( which is part of the standard usage guide for this playbook )

##### Checklist

- [X] commit message has one of the [standard prefixes]
- [X] other documentation is changed or added (if applicable)
- [X] VPC/QPC not applicable for this PR

### CODE REVIEW

**Comment by steelhead31** [comment]
My nagios linter no longer fails :)

**Comment by karianna** [comment]
Looks like some ansible task naming errors to go?

File: ansible/playbooks/nagios/play_setup_server.yml
Comment (karianna) [inline ansible/playbooks/nagios/play_setup_server.yml:5]: commented out code?

**Review by karianna — CHANGES_REQUESTED** [review]

File: ansible/playbooks/nagios/play_setup_server.yml
Comment (steelhead31) [reply ansible/playbooks/nagios/play_setup_server.yml:5]: Yes, in order to stop the linter complaining, I need to remove the encrypted ansible vault, secrets_setup_server.enc .. however I want to leave the reference to it, and have it refer to the document.. I'll expand the comments to cover reading the docs (which details creating the file, before running the playbook ).

**Review by Haroon-Khel — APPROVED** [review]
The remaining linter errors are not related to this pr. LGTM

**Review by sxa — COMMENTED** [review]
I'm ok with this but sine @karianna had put a block on it I'd prefer to give him the chance to approve before merging.

**Review by karianna — APPROVED** [review]

**Pull request merged**
